CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.1
+
+### Added
++ `Api256.hash256(hashable_buffer: Buffer): Buffer;`
```

Comment:
I'm being really nitpicky here, but since we can't communicate it via the types, it might be nice to explain that the Buffer you get back from this method will always be 32 bytes in length.